wadas/ui/configure_ftp_cameras_dialog.py

```diff
@@ -164,9 +164,11 @@ def accept_and_close(self):
                                         f"WADAS_FTP_camera_{camera.id}", ""
                                     )
                                     if credentials and (
-                                            credentials.username != cur_user
-                                            or credentials.password != cur_pass
+                                        credentials.username == cur_user
+                                        or credentials.password == cur_pass
```

Comment:
I think that here you should only check for the current user and not for the password. The user could be the same, but the password might have changed -> update the keyring.